packages/trie/test/proof.spec.ts

```diff
@@ -143,3 +154,283 @@ describe('simple merkle proofs generation and verification', () => {
     assert.equal(bytesToUtf8(val!), 'c')
   })
 })
+
+describe('createRangeProof()', function () {
```

Comment:
Any particular reason to add this test here and not in `test/proof/range.spec.ts`?

Reply:
No reason. This PR is rather dirty. It should be cleaned up :)